app/models/UtilNumbers.scala

```diff
@@ -0,0 +1,21 @@
+package models
+
+/**
+  * Contains util functions around numbers.
+  */
+object UtilNumbers {
+
+  /**
+    * Convert string to optional int.
+    * If s cannot be converted to Int, this method returns None
+    * @param s the string to convert
+    * @return
+    */
+  def toInt(s: String): Option[Int] = {
+    try {
+      Some(s.toInt)
+    } catch {
+      case e: Exception => None
+    }
+  }
+}
```

Comment:
I would suggest using one of the suggestions from [stack overflow](http://stackoverflow.com/questions/23811425/scala-convert-string-to-int-or-none) instead